File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/DefaultProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports System

Namespace Aspose.Tasks.Examples.VisualBasic.WorkingWithProjects
    Public Class DefaultProperties
        Public Shared Sub Run()
            'ExStart: DefaultProperties
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Projects()

            Dim prj__1 As New Project(dataDir & "DefaultProperties.mpp")

            'Display default properties
            'Display default properties
            Console.WriteLine("New Task Default Start: " + prj__1.[Get](Prj.DefaultStartTime).ToShortDateString())
            Console.WriteLine("New Task Default Type:" + prj__1.[Get](Prj.DefaultTaskType))
            Console.WriteLine("Resouce Default Standard Rate: " + prj__1.[Get](Prj.DefaultStandardRate).ToString())
            Console.WriteLine("Resource Default Overtime Rate: " + prj__1.[Get](Prj.DefaultOvertimeRate).ToString())
            Console.WriteLine("Default Task EV Method: " + prj__1.[Get](Prj.DefaultTaskEVMethod).ToString())
            Console.WriteLine("Default Cost Accrual: " + prj__1.[Get](Prj.DefaultFixedCostAccrual).ToString())
            'ExEnd: DefaultProperties
        End Sub
    End Class
End Namespace

File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/DefaultProperties.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Tasks. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Tasks;
using System;

namespace CSharp.Projects
{
    public class DefaultProperties
    {
        public static void Run()
        {
            //ExStart: DefaultProperties
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();

            Project prj = new Project(dataDir+ "DefaultProperties.mpp");

            //Display default properties
            Console.WriteLine("New Task Default Start: " + prj.Get(Prj.DefaultStartTime).ToShortDateString());
            Console.WriteLine("New Task Default Type: " + prj.Get(Prj.DefaultTaskType));
            Console.WriteLine("Resouce Default Standard Rate: " + prj.Get(Prj.DefaultStandardRate).ToString());
            Console.WriteLine("Resource Default Overtime Rate: " + prj.Get(Prj.DefaultOvertimeRate).ToString());
            Console.WriteLine("Default Task EV Method: " + prj.Get(Prj.DefaultTaskEVMethod).ToString());
            Console.WriteLine("Default Cost Accrual: " + prj.Get(Prj.DefaultFixedCostAccrual).ToString());
            //ExEnd: DefaultProperties
        }
    }
}